src/functions/lever.js

```diff
@@ -0,0 +1,67 @@
+const got = require('got')
+const { parse, format } = require('url')
+const isEqual = require('lodash.isequal')
+
+/**
+ *
+ * This lambda is used to check for any differences in roles listed on yld.io
+ * and our lever account. If any new changes are found we trigger a new site deployment.
+ *
+ * The yld.io/meta.json endpoint is created on the postBuild method of gatsby.
+ * See gatsby-node.js for more detials.
+ *
+ * I've not found a way to test this in a deploy preview as we have no reference to the
```

Comment:
Not sure about the `I've not found` as new joiners don't know who wrote this if not using git blame but it's a minor thing

Reply:
Oh, I sometimes hate the emoji `+1` ends up being `thumbs up`.

Anyway, I agree that putting name and date would be helpful for posterity.

Reply:
@GBouffard does the author matter? Can just as well write `We've not found a way...` & have that be talking from the perspective of "the team building this website"

Reply:
I mean, my knowledge of the project evaporates pretty quickly after the project ends, but I am able to pick stuff up, and remember the bits that took actual research. "The team" is great if you have a stable group, but here it's very fluid.